release note (#2234)
SumoLogic/sumologic-documentation: blog-service/2023-04-28-apps-c2c.md

---

We're excited to announce the release of the new Proofpoint on Demand app for Sumo Logic. This app leverages Sumo Logic's [Cloud-to-Cloud Proofpoint on Demand source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/proofpoint-on-demand-source) to provide an improved security posture by analyzing message logs. Here are some of the key features that this app offers:

- **Email security monitoring**. Monitor message traffic to detect and prevent spam, phishing, and other email-borne threats.
- **Compliance monitoring**. Monitor email communications for compliance with internal policies and external regulations such as Data Loss Prevention (DLP), Domain-based Message Authentication, Reporting and Conformance (DMARC), and other relevant regulations.
- **Incident investigation**. Quickly investigate potential security incidents by searching and analyzing email security and compliance data. This includes identifying the source of a security threat and the extent of its impact.
- **User behavior monitoring**. Monitor behavior related to email communication to identify potential insider threats or unauthorized access.

> Check out our technical documentation [here](/docs/integrations/saas-cloud/proofpoint-on-demand) to learn how you can set up and use the proofpoint-on-demand app for Sumo Logic.

Reviewer: ```suggestion
For more information, check out our technical documentation [here](/docs/integrations/saas-cloud/proofpoint-on-demand).
```
